main.tf

```diff
@@ -628,11 +643,6 @@ resource "aws_elastic_beanstalk_environment" "default" {
     name      = "BatchSize"
     value     = "1"
   }
-  setting {
-    namespace = "aws:elasticbeanstalk:command"
-    name      = "DeploymentPolicy"
```

Comment:
Why are you dropping this?

Reply:
It already exists. For some reason, it was set twice.

https://github.com/cloudposse/terraform-aws-elastic-beanstalk-environment/blob/master/main.tf#L389-L393

```
  setting {
    namespace = "aws:elasticbeanstalk:command"
    name      = "DeploymentPolicy"
    value     = "${var.rolling_update_type == "Immutable" ? "Immutable" : "Rolling"}"
  }
```

and

```
  setting {
    namespace = "aws:elasticbeanstalk:command"
    name      = "DeploymentPolicy"
    value     = "Rolling"
  }
```